src/coreclr/runtime.proj

```diff
@@ -50,6 +50,8 @@
       <_CoreClrBuildArg Condition="'$(HostCrossOS)' != ''" Include="-hostos $(HostCrossOS)" />
       <_CoreClrBuildArg Include="-outputrid $(OutputRID)" />
       <_CoreClrBuildArg Condition="'$(BuildSubdirectory)' != ''" Include="-subdir $(BuildSubdirectory)" />
+      <_CoreClrBuildArg Include="-cmakeargs &quot;-DCLR_DOTNET_HOST_PATH=$(DOTNET_HOST_PATH)&quot;" Condition="'$(DOTNET_HOST_PATH)' != ''" />
+      <_CoreClrBuildArg Include="-cmakeargs -DCLR_DOTNET_HOST_PATH=dotnet" Condition="'$(DOTNET_HOST_PATH)' == ''" />
```

Comment:
I think only .NET Framework doesn't have DOTNET_HOST_PATH set, which is not supported by runtime.proj. Did you notice build failing somewhere without this line?

Reply:
No, I saw `eng/illink.targets`, but I realize now it's talking about .NET Framework. I'll remove this.